src/sentry/auth_v2/endpoints/auth_merge_user_accounts.py

```diff
@@ -13,7 +15,7 @@
 from sentry.users.models.user_merge_verification_code import UserMergeVerificationCode
 
 
-class AuthMergeUserAccountsValidator(serializers.Serializer):
+class AuthMergeUserAccountsValidator(serializers.Serializer[Any]):
```

Comment:
this should be `Never` rather than `Any` (as we do not use the actual object serializer parts of this -- just an extracted dict)